### DISCUSSION TITLE
👋 Welcome to our GitHub Discussion!

### DISCUSSION THREAD

**Discussion by zheeeng** [galacean/effects-specification#1] [Announcements]
## 👋 欢迎来到我们的 GitHub Discussion！

这是一个讨论和交流的空间，无论您是新手还是经验丰富的开发者，都可以在这里找到解答或与其他人交流。

我们非常欢迎您在这里：
* 提出对项目的改进建议或功能请求；
* 分享您的想法、知识与学习资源；
* 与 Galacean Effects 的开发成员、大众设计师、动效开发者互动。

如果遇到了 Galacean Effects Specification 相关的问题，欢迎移步 Issue 区，将会有相关的人员进行解答。

请在发帖前注意一些事项：

* 尽量使用简洁的标题，以便其他人快速了解您的问题或主题。
* 尊重他人的意见和观点，避免使用侮辱性、攻击性或令人不悦的语言。如果有争议，请友好地讨论。
* 避免在讨论中散布垃圾信息、政治相关内容、招聘消息、广告或其他无关内容。

感谢您的参与和贡献！我们期待与您一起建立一个开放、友好和有益的讨论社区。

---

## 👋 Welcome to our GitHub Discussion!
This is a space for discussion and exchange, where both newcomers and experienced developers can find answers or interact with others.

We are very glad to have you here to:

* Suggest improvements or feature requests for the project;
* Share your ideas, knowledge, and learning resources;
* Interact with Galacean Effects developers, general designers, and motion effects developers. 

If you encounter any issues related to Galacean Effects Specification, feel free to head over to the Issues section, where relevant personnel will assist you.

Please keep in mind some points before posting:

* Try to use concise titles so that others can quickly understand your question or topic.
* Respect others' opinions and viewpoints, avoid using insulting, aggressive, or displeasing language. If there are disagreements, please discuss politely.
* Avoid spreading spam, political content, job listings, advertisements, or other irrelevant content in the discussions.

Thank you for your participation and contribution! We look forward to building an open, friendly, and beneficial discussion community with you.

**Comment by HypnosNova** [comment]
Why does galacean-effect not fit galcean engine well?

It took Ant-Group such a long time developing  engine-mars to help developers using  galacean-effect animations in galacean engine. But it is so difficult to use because of two different rendering pipeline and some hack behaviors. Many bugs occerred when rendering complex animations and hierarchical nesting. And two pipelines make JS libraries very big.

I think you should let the technology much easier for developers to use, not only designers.

**Reply by zheeeng** [reply]
Thank you for bringing your concerns to our attention.

Regarding the compatibility issues between Galacean Effects and Galacean Engine, we understand the importance and are planning to solve it in the near future. Perhaps we can discuss implementing a collaborative mechanism to expedite the process?

Regarding your second question, we recognize that both developers and designers play crucial roles. We are committed to meeting the needs of both, and our focus may vary at different stages of our product development. We have made significant improvements to the developer experience in the latest release of Galacean Effects Runtime v1.0.0, which we detailed at the 1124 product launch event.

We acknowledge any existing issues and are dedicated to continuous improvement. We are embracing the community to help us improve. We also encourage and welcome more feedback from users. If you encounter specific issues, please feel free to engage in discussions and collaborations in the issue section.
